README.md

```diff
@@ -126,3 +130,45 @@ logging best-practices
     - "info"
 - `title` (string) - the event that occurred
 - `data` (object) - other parameters describing the event
+
+### Middleware
+
+Kayvee includes logging middleware, compatible with expressJS.
+
+The middleware can be added most simply via
+
+```js
+kayveeMiddleware = require('kayvee/middleware')
+
+var app = express()
+app.use(kayveeMiddleware())
+```
+
+It also supports user configuration via an `options` object.
+It prints the values of the headers or the results of the handlers.
+If a value is `undefined`, the key will not be printed.
```

Comment:
@agclever does that seem like it would block any use cases you were thinking of? I don't want to be unnecessary strict, just concerned that e.g. infra-needed params for metrics dashboards could get overridden accidentally by a user